modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss

```diff
@@ -48,6 +48,13 @@ body {
 	font-size: var(--font-size-base);
 }
 
+#wrapper {
```

Comment:
can't you just do `body, #wrapper`?

Reply:
![](https://media.giphy.com/media/6yRVg0HWzgS88/giphy.gif)

Reply:
![](https://media.giphy.com/media/fdUIa3WZAdIsLKCjDB/giphy.gif)

Reply:
fixed!